subiquity/tests/api/test_api.py

```diff
@@ -102,14 +102,15 @@ async def server_shutdown(self, immediate=True):
         except aiohttp.client_exceptions.ServerDisconnectedError:
             return
 
-    async def spawn(self, socket_path, machine_config, bootloader='uefi'):
+    async def spawn(self, tempdir, socket, machine_config, bootloader='uefi'):
```

Comment:
Maybe `output_base` is better than `tempdir` in this context? I don't know

Reply:
I like that better, will do